### PR TITLE
Add assets for 'TRO-94c925'

### PR DESCRIPTION
**Pull request by NeltudOne**

### CODE REVIEW

File: tokens/TRO-94c925/info.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-	"website": "https://nelsontuduriprods.wixsite.com/tableauxnelsontuduri",
-	"description": "Original token designed for artistic NFTs and physical artworks.",
-	"social": {
-		"twitter": "https://twitter.com/NTuduriProd"
```
Comment (NeltudOne) [inline tokens/TRO-94c925/info.json:5]: twitter deleted/suspended

File: tokens/TRO-94c925/info.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-	},
-	"status": "active"
```
Comment (NeltudOne) [inline tokens/TRO-94c925/info.json:7]: make inactive

File: tokens/TRO-94c925/info.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-}
+    "website": "https://nelsontuduriprods.wixsite.com/tableauxnelsontuduri",
+    "description": "Original token designed for artistic NFTs and physical artworks.",
+    "status": "active",
+    "social": {
+        "twitter": "https://twitter.com/tudurioriginal"
```
Comment (NeltudOne) [inline tokens/TRO-94c925/info.json:6]: make active please

**Comment by NeltudOne** [comment]
Can you change twitter to tudurioriginal please and linktree too

**Comment by paulmultiversx** [comment]
Hi  @NeltudOne.

Please follow the guidelines to maximize the chances of passing through the filters, which are in place to create a healthy and sustainable token ecosystem: https://multiversx.com/blog/branding-esdt-tokens-nft-collections/

You can create a new PR once you've launched a proper website & you've achieved at least 1000+ Twitter, Discord or TG official channel followers.

Thank you.

**Pull request closed without merging**